spec/features/idv/steps/forgot_password_step_spec.rb

```diff
@@ -36,4 +36,45 @@
 
     expect(current_path).to eq edit_user_password_path
   end
+
+  context 'with idv app feature enabled', js: true do
+    before do
+      allow(IdentityConfig.store).to receive(:idv_api_enabled_steps).
+        and_return(['password_confirm', 'personal_key', 'personal_key_confirm'])
+    end
+
+    it 'goes to the forgot password page from the review page' do
+      start_idv_from_sp
+      complete_idv_steps_before_review_step
+
+      click_button t('idv.forgot_password.link_text')
+
+      expect(page.current_path).to eq(idv_app_forgot_password_path)
+    end
+
+    it 'goes back to the review page from the forgot password page' do
+      start_idv_from_sp
+      complete_idv_steps_before_review_step
+
+      click_button t('idv.forgot_password.link_text')
+      click_button t('idv.forgot_password.try_again')
+
+      expect(page.current_path).to eq "#{idv_app_path(step: :password_confirm)}/"
```

Comment:
Have to add the trailing slash here, since that's how the client-side routing currently [behaves](https://github.com/18F/identity-idp/blob/32ff2e9d4dbe994bb6dc46dd48c35ccdd2bb625c/app/javascript/packages/form-steps/use-history-param.ts#L48-L49) (recently revised in #6374). I'm looking to change this again as part of #6387 so that we won't need to do this ([strip the trailing slash](https://github.com/18F/identity-idp/blob/a5c5966b60711b0854d34e259046493dadf4c436/app/javascript/packages/form-steps/use-history-param.ts#L20-L22)).